### DISCUSSION TITLE
How to do an incremental learning where new items appears gradually?

### DISCUSSION THREAD

**Discussion by lmatejka** [NVIDIA-Merlin/Transformers4Rec#790] [Q&A]
I suppose that NVT workflow process whole dataset and set cardinatlity and labels of item_ids and category_ids? When I want to do a continuous learning day by day, I can train model on new interactions from that day, but when new item_id arrives, no label is present in NVT workflow. What is a proper solution for that? I have to perform new NVT transformation on whole dataset with all items included?

**Comment by Tottowich** [comment]
Hi! @lmatejka 
Did you find any solution to this?

**Reply by lmatejka** [reply]
Not really, i postponed this and whole activity stays only in PoC. Let me know if you find out how to tackle this.
